docs/Documentation/model_introduction.md

```diff
@@ -10,22 +10,47 @@
    omz_tools_downloader
 
 
-Every deep learning workflow begins with obtaining a model. You can choose to prepare a custom one, use a ready-made solution and adjust it to your needs, or even download and run a pre-trained network from an online database, such as OpenVINO's :doc:`Open Model Zoo <model_zoo>`.
+Every deep learning workflow begins with obtaining a model. You can choose to prepare a custom one, use a ready-made solution and adjust it to your needs, or even download and run a pre-trained network from an online database, such as `TensorFlow Hub <https://tfhub.dev/>`__, `Hugging Face <https://huggingface.co/>`__, `Torchvision models <https://pytorch.org/hub/>`__.
 
-:doc:`OpenVINO™ supports several model formats <Supported_Model_Formats>` and allows to convert them to it's own, OpenVINO IR, providing a tool dedicated to this task.
+:doc:`OpenVINO™ supports several model formats <Supported_Model_Formats>` and allows converting them to it's own, `openvino.runtime.Model <api/ie_python_api/_autosummary/openvino.runtime.Model.html>`__ or `ov.Model <api/ie_python_api/_autosummary/openvino.runtime.Model.html>`__, providing a tool dedicated to this task.
```

Comment:
`openvino.runtime.Model` is the same as `ov.Model`, the latter is because we usually `import openvino.runtime as ov`. Please do not make the impression that they are two different things.

Reply:
> `openvino.runtime.Model` is the same as `ov.Model`, the latter is because we usually `import openvino.runtime as ov`. Please do not make the impression that they are two different things.

```suggestion
:doc:`OpenVINO™ supports several model formats <Supported_Model_Formats>` and allows converting them to it's own, `openvino.runtime.Model <api/ie_python_api/_autosummary/openvino.runtime.Model.html>`__ (`ov.Model <api/ie_python_api/_autosummary/openvino.runtime.Model.html>`__ ), providing a tool dedicated to this task.
```